lib/BackgroundJob/ExpireGroupTrash.php

```diff
@@ -50,6 +50,7 @@ public function __construct(
 		$this->trashBackend = $trashBackend;
 		$this->expiration = $expiration;
 		$this->config = $config;
+		$this->setTimeSensitivity(self::TIME_SENSITIVE);
```

Comment:
this is the default

if you think this should only run at night in the maintenance window, you need to set it to "TIME_INSENSITIVE"

Reply:
🙃